### PR TITLE
fix(copy): rename project management to issue tracking

### PR DESCRIPTION
**Pull request by robinrendle**
Copy fixes related to [this PR](https://github.com/getsentry/sentry/pull/32910).

## Before
CleanShot 2022-03-24 at 10 43 56

## After
CleanShot 2022-03-24 at 10 44 36

### CODE REVIEW

File: src/docs/product/integrations/index.mdx [hunk 1]
```diff
@@ -21,66 +21,66 @@ For more details, see the [full Integration Platform documentation](/product/int
...
-| Integration                                               | Commit Tracking | Suspect Commits | Resolve via Commit/PR |
-| --------------------------------------------------------- | --------------- | --------------- | --------------------- |
+| Integration                                                                | Commit Tracking | Suspect Commits | Resolve via Commit/PR |
+| -------------------------------------------------------------------------- | --------------- | --------------- | --------------------- |
 | [Azure DevOps](/product/integrations/source-code-mgmt/azure-devops/)       | X               | X               | X                     |
 | [Bitbucket](/product/integrations/source-code-mgmt/bitbucket/)             | X               | X               | X                     |
 | [GitHub/GitHub Enterprise](/product/integrations/source-code-mgmt/github/) | X               | X               | X                     |
 | [GitLab](/product/integrations/source-code-mgmt/gitlab/)                   | X               | X               | X                     |
 
-## Project Management
```
Comment (robinrendle) [inline src/docs/product/integrations/index.mdx:46]: This is the only change I made, everything else is automatic.

File: src/docs/product/integrations/index.mdx [hunk 2]
```diff
@@ -21,66 +21,66 @@ For more details, see the [full Integration Platform documentation](/product/int
...
-| [GitLab (Issues)](/product/integrations/source-code-mgmt/gitlab/)                 | X                      |            |
-| [Jira/Jira Server](/product/integrations/project-mgmt/jira/)                  | X                      | X*         | X
-| [Linear](/product/integrations/project-mgmt/linear/)                          | X                      |            |
-| Phabricator                                                      | X                      |            |
-| Pivotal Tracker                                                  | X                      |            |
-| Redmine                                                          | X                      |            |
-| [Shortcut](/product/integrations/project-mgmt/shortcut/)                    | X                      |            |
-| [Teamwork](/product/integrations/project-mgmt/teamwork/)                      | X                      |            |
-| Trello                                                           | X                      |            |
+## Issue Tracking
```
Comment (imatwawana) [inline src/docs/product/integrations/index.mdx:46]: Adding this to help with heading redirects

```suggestion
## Issue Tracking {#project-management}
```

File: src/docs/product/integrations/integration-platform/ui-components/index.mdx [hunk 3]
```diff
@@ -343,7 +343,7 @@ When creating or linking an issue, the response format _must_ have the following
 
 When an external issue is created or linked in Sentry, Sentry shows a display name that links back to the service where it was either created or linked. The display name is composed of two pieces: the `project` and the `identifier`. A hash (#) connects each piece. Here's an example of what it looks like in Sentry:
 
-![Sentry's sidebar button illustrating that an issue is linked.](../../project-mgmt/shortcut/shortcut_story_in_linked_issues_82.png)
+![Sentry's sidebar button illustrating that an issue is linked.](../../issue-tracking/shortcut/shortcut_story_in_linked_issues_82.png)
 
 # Supplementary Information
```
Comment (imatwawana) [inline src/docs/product/integrations/integration-platform/ui-components/index.mdx:348]: Fixing heading level - anything in the page has to start at level 2
```suggestion
## Supplementary Information
```

File: src/docs/product/integrations/issue-tracking/index.mdx [hunk 4]
```diff
@@ -0,0 +1,22 @@
+---
+title: Issue Tracking
+sidebar_order: 40
+description: "Learn more about Sentry's issue tracking integrations."
```
Comment (imatwawana) [inline src/docs/product/integrations/issue-tracking/index.mdx:4]:
```suggestion
description: "Learn more about Sentry's issue tracking integrations."
redirect_from: 
- /product/integrations/project-mgmt/
```

**Review by imatwawana — COMMENTED** [review]
Redirects need to be added on the child pages of Issue Tracking, so I'm gonna commit the changes I've suggested and pull in the PR to add those updates.

**Review by imatwawana — APPROVED** [review]
Everything looks like it's working now

**Pull request merged**